third_party/lua/lunix.c

```diff
@@ -3196,6 +3199,71 @@ static int LuaUnixFdopendir(lua_State *L) {
   }
 }
 
+
+// unix.mount(source:str, target:str, filesystemtype: str[, mountflags: int[, data: str]])
+//     ├─→ true
+//     ├─→ nil, unix.Errno
+static int LuaUnixMount(lua_State *L) {
+  int olderr = errno;
+  return SysretBool(
+    L, "mount", olderr,
+    mount(luaL_checkstring(L, 1), luaL_checkstring(L, 2), luaL_checkstring(L, 3),
+        luaL_optinteger(L, 4, 0), luaL_optstring(L, 5, NULL)));
+}
+
+// unix.umount(target: str[, flags: int)
+//     ├─→ true
+//     ├─→ nil, unix.Errrno
+static int LuaUnixUnmount(lua_State *L) {
+  int olderr = errno;
+  return SysretBool(
+    L, "unmount", olderr,
+    unmount(luaL_checkstring(L, 1), luaL_optinteger(L, 2, 0)));
+}
+
+// unix.prctl(option: int[, arg2: int[, arg3: int[, arg4: int[, arg5: int]]]])
+//     ├─→ result: int
+//     └─→ nil, unix.Errno
+static int LuaUnixPrctl(lua_State *L) {
```

Comment:
None of the get-like prctl variants are going to work, as there is no way to get the values back to the Lua layer. These values either need to be returned are prctl results (it seems like most of them are stored as arg2, so may be need to be returned as a string or number or need to be taken as one table parameter, in which case the table can be modified to pass the values back. The first option is going to be more consistent with how the rest of unix.* functions operate.

I'm not familiar with the implementation details of prctl to access its coverage in cosmopolitan, but I'm sure @jart can comment on that.